experimental/examples/ConfigExample.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../../vendor/autoload.php';
+
+use OpenTelemetry\Experimental\Config\ConfigBuilder;
+use OpenTelemetry\Experimental\Config\Exporter\NewRelic\Config as NewRelicConfig;
+use OpenTelemetry\Experimental\Config\Exporter\Zipkin\Config as ZipkinConfig;
+
+//@see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md
+putenv('OTEL_RESOURCE_ATTRIBUTES=service.version=1.0.0,service.namespace=my_app');
+putenv('OTEL_SERVICE_NAME=example-app');
+putenv('OTEL_LOG_LEVEL=warning');
+putenv('OTEL_TRACES_SAMPLER=traceidratio');
+putenv('OTEL_TRACES_SAMPLER_ARG=0.95');
+putenv('OTEL_TRACES_EXPORTER=otlp,zipkin,newrelic');
+putenv('OTEL_EXPORTER_ZIPKIN_ENDPOINT=http:zipkin/v1/traces');
+putenv('OTEL_EXPORTER_ZIPKIN_TIMEOUT=99');
+putenv('OTEL_PHP_TRACES_PROCESSOR=batch,simple,noop');
+putenv('OTEL_BSP_SCHEDULE_DELAY=10000');
+putenv('OTEL_ATTRIBUTE_COUNT_LIMIT=111');
+putenv('OTEL_PROPAGATORS=tracecontext');
+
+echo 'Creating Config From Environment and user config' . PHP_EOL;
```

Comment:
What is "user config"? are env vars not provided by users?

Reply:
True, they are. I could merge them all together, but haven't because it requires sdk->contrib references - at least, the way I'm thinking of it does. As mentioned in an earlier comment, this might be resolved by either having that logic sit in `contrib`, or even removing env vars from feature and keeping that knowledge where it is now, in the various constructors.